cmake/OpenCVDetectCUDA.cmake

```diff
@@ -88,8 +88,28 @@ if(CUDA_FOUND)
       set(__cuda_arch_bin "3.2")
       set(__cuda_arch_ptx "")
     elseif(AARCH64)
-      set(__cuda_arch_bin "5.3")
-      set(__cuda_arch_ptx "")
+      message(STATUS "Determining Jetson Revision")
+      # Detemine whether the platform is Jetson TX1 or TX2
+      try_run(_jetson_result _jetson_compile
+              "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/"
+              "${OpenCV_SOURCE_DIR}/cmake/checks/OpenCVDetermineJetson.cpp")
+      message(STATUS "JETSON_RESULT : ${_jetson_result}")
+      message(STATUS "JETSON_COMPILE : ${_jetson_compile}")
```

Comment:
Right. Thanks for your suggestion.
I always forget about the cross compilation
I'll remove the two lines of ```message``` but I'm not sure how to detect without using ```try_run```